src/interactivity/win32/ut_interactivity_win32/UiaTextRangeTests.cpp

```diff
@@ -1346,4 +1361,312 @@ class UiaTextRangeTests
             VERIFY_SUCCEEDED(utr->ScrollIntoView(alignToTop));
         }
     }
+
+    TEST_METHOD(GetAttributeValue)
+    {
+        Log::Comment(L"Check supported attributes");
+        IUnknown* notSupportedVal;

```

Comment:
Make sure you're treating this COM pointer correctly.

In general: If you find yourself with a raw `IWhatever*`, you should be double-checking your math every time.